electricpy/__init__.py

```diff
@@ -5077,4 +5077,50 @@ def pi_attenuator(Adb, Z0):
     R2 = (Z0/2)*(_np.power(10, x) - (1/(_np.power(10, x))))
 
     return R1,R2
+
+
+def lm317(r1, r2, v_out):
+    r"""
+    LM317.
+
+    The LM317 is a linear voltage regulator that can be adjusted to supply a specific output voltage.
```

Comment:
If we're going to directly quote the BasicTables content, we should give some attribution credit.

for e.g.,

```
The LM317 is a linear voltage regulator that can be adjusted to supply a specific output voltage.
The LM317 has three pins, adjust, output and input. The LM317 is often connected as in the image below. [1]_

.. [1] Electronial, "LM317" BasicTables, Accessed May, 2022
   https://www.basictables.com/electronics/lm317
```

In retrospect, this should probably be done across the board, but it's especially important in this case since this is a direct quote.

Reply:
Fixing them